app/resources/scanned_resources/scanned_resource_change_set.rb

```diff
@@ -59,4 +59,8 @@ def primary_terms
       :append_id
     ]
   end
+
+  def preserve?
+    Array.wrap(resource.preservation_policy).include?("cloud") && state == "complete"
+  end
```

Comment:
Given the decisions we made today (not preserving linked Collections or EphemeraTerms), I think we probably want almost every kind of resource to allow preservation (everything but FileSets?).

Reply:
Yeah, this was just to get some tests passing.